src/CattyTests/SceneTests.swift

```diff
@@ -123,4 +123,40 @@ final class SceneTests: XCTestCase {
         XCTAssertEqual(0, userVariable.textLabel?.text?.count)
         scene.stopProject()
     }
+
+    func testUpdate() {
+
+        class DummyCBSpriteNode: CBSpriteNode {
+
+            var methodCallCount = 0
+
+            override func update(_ currentTime: TimeInterval) {
```

Comment:
Please incorporate your changed to the existing _CBSpriteNodeMock_ so that they can be reused.